### DISCUSSION TITLE
✨ Run your GitHub Actions locally in VS Code!

### DISCUSSION THREAD

**Discussion by SanjulaGanepola** [microsoft/vscode-discussions#2440] [Extension Show and Tell]
Hey everyone! I recently have been working on this cool new VS Code extension and wanted to share it here for the world to see. I'm always open to feedback and new ideas!

Run your GitHub Actions locally with the power of the [GitHub Local Actions](https://marketplace.visualstudio.com/items?itemName=SanjulaGanepola.github-local-actions) VS Code extension! Say goodbye to the hassle of committing and pushing changes every time you need to test a workflow. This extension lets you quickly and efficiently run your workflows locally, leveraging the [nektos/act](https://github.com/nektos/act) CLI tool. Enjoy a seamless experience with an interface designed to feel as familiar as the official [GitHub Actions](https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-github-actions) extension.

* 🚀 **Run Workflows/Jobs**: Run entire workflows or specific jobs locally without leaving your editor.
* ⚡ **Trigger Events**: Trigger standard [GitHub events](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows) to run multiple workflows.
* 📖 **View Workflow Run History**: Track and investigate past workflow logs.
* ⚙️ **Manage Workflow Settings**: Define secrets, variables, inputs, and runners to be used when executing workflows.

**Install**: https://marketplace.visualstudio.com/items?itemName=SanjulaGanepola.github-local-actions
**GitHub Repository**: https://github.com/SanjulaGanepola/github-local-actions

![GitHub Local Actions](https://raw.githubusercontent.com/SanjulaGanepola/github-local-actions/main/images/github-local-actions.gif)

**Comment by mxro** [comment]
Love it, this works very well!

**Reply by SanjulaGanepola** [reply]
Thank you, that's great to hear!

**Comment by wiltonlpolescadesouza-eaton** [comment]
Is it possible to use Podman as a replacement for Docker Desktop?
